packages/widgets/src/components/CollapsibleNavBar/CollapsibleNavBar.tsx

```diff
@@ -0,0 +1,153 @@
+import { useLocation } from 'react-router-dom';
+import * as S from './styles';
+
+import {
+  Accordion,
+  AccordionGroup,
+  AccordionGroupContextProvider,
+  AccordionTitle,
+  ResponsiveImage,
+  Wrapper,
+} from '@ssa-ui-kit/core';
+
+import CollapsibleNavBarBase from './CollapsibleNavBarBase';
+import CollapsibleNavBarWrapper from './CollapsibleNavBarWrapper';
+import CollapsibleNavBarList from './CollapsibleNavBarList';
+import CollapsibleNavBarItem from './CollapsibleNavBarItem';
+import CollapsibleNavBarLink from './CollapsibleNavBarLink';
+import CollapsibleNavToggle from './CollapsibleNavToggle';
+
+import { INavBarExtendedProps } from './types';
+import { NavBarAccordionContent } from './NavBarAccordionContent';
+import { CollapsibleNavBarPopover } from './NavBarPopover';
+import { NavContentToggle } from './CollapsibleNavContentToggle';
+import { TriggerIcon } from './TriggerIcon';
+
+/**
+ * UI Component that shows the collapsible navigation bar
+ *
+ */
+export const CollapsibleNavBar = ({ items }: INavBarExtendedProps) => {
+  const { pathname } = useLocation();
+
+  return (
+    <CollapsibleNavBarBase>
+      <CollapsibleNavToggle />
+
+      <CollapsibleNavBarWrapper>
+        <Wrapper css={S.LogoWrapper}>
+          <ResponsiveImage
+            css={S.ResponsiveLogo}
+            srcSet="https://firebasestorage.googleapis.com/v0/b/admin-themes.appspot.com/o/logo%2FCTP_Large_Left.png?alt=media&token=b6fe7ab8-fd0b-475f-bb08-360311f27693 69w, https://firebasestorage.googleapis.com/v0/b/admin-themes.appspot.com/o/logo%2FCTP_Medium_Left.png?alt=media&token=a1aeba69-7c07-40c9-aeac-c2477640870d 55w"
+            sizes="(min-width: 900px) 55px, (min-width: 1440px) 69px"
+            src="https://firebasestorage.googleapis.com/v0/b/admin-themes.appspot.com/o/logo%2FCTP_Small_Left.png?alt=media&token=bff7149e-3b90-4657-8a11-040e83990e6f"
+            alt="SSA CTP logo"
+          />
+          <NavContentToggle id={'contentToggler'} />
+        </Wrapper>
+        <CollapsibleNavBarList>
+          {items.map((item) => {
+            if ('items' in item) {
```

Comment:
I would split this `if/else` into multiple files. To make the code more readable.